README.md

```diff
@@ -1,3 +1,15 @@
+![Leanplum](Leanplum.png)
+
+<p align="center">
+<img src='https://jenkins.leanplum.com/buildStatus/icon?job=apple-sdk-github'></a>
```

Comment:
yeah, this one is triggered on PR, we need the one that is showing status of last merged PR to show the current state of the develop branch.